astor/__init__.py

```diff
@@ -19,7 +20,9 @@
 from .op_util import symbol_data  # NOQA
 from .tree_walk import TreeWalk  # NOQA
 
-__version__ = '0.8.0'
+ROOT = os.path.dirname(__file__)
+with open(os.path.join(ROOT, 'VERSION')) as version_file:
+    __version__ = version_file.read().strip
```

Comment:
Should be strip()

Reply:
hmmm, I knew this too.. I guess it worked, just not in the intended way:

```
>>> import astor
>>> astor.__version__
<built-in method strip of str object at 0x7f8543f60930>
```

Reply:
done, sorry about that :/